mantidproject/3rdpartysources
# Add cmake support

This PR adds cmake support. It uses a cmake custom target to run f2py, outputing all the libraries in the build/bin folder.

To test:

- Install miniconda

- conda create --name fortran python=3.8
- conda activate fortran
- conda install numpy
-  On windows do: conda install -c msys2 m2w64-gcc-libgfortran 
- On osx and Linux do:  conda install -c conda-forge fortran-compiler
- mkdir build
- cd build
- cmake ..
- cmake --build .

The generated python modules should be in the folder build/bin. Open a python terminal in that folder and test each can be imported.
